build.py

```diff
@@ -1,75 +1,180 @@
 # A nasty basic implementation with no tests to preserve the ability to build data required by the
 # service.
 
-import glob
+import argparse
 import json
+import logging
+import logging.config
 import pathlib
-import sys
-
-OUTPUT = "./build"
-
-VERSION = sys.argv[1]
-
-
-def load_json_file(src_path):
-    with open(str(src_path)) as fd:
-        return json.load(fd)
-
-
-def expand_remote_config_source(config_source, glob_list_keys):
-    config = {}
-    config["version"] = VERSION
-    for key in glob_list_keys:
-        config[key] = expand_glob_list(config_source[key])
-    return config
-
-
-def expand_glob_list(globs):
-    results = []
-    for g in globs:
-        for path in glob.glob(g):
-            config_part = load_json_file(path)
-            results.append(config_part)
-    return results
-
-
-def produce_v2_config(src_path, output_dir):
-    config_source = load_json_file(src_path)
-    config = expand_remote_config_source(
-        config_source, ["conditional_gathering_rules", "container_logs"]
+import shutil
+
+import git
+import semver
+
+logger = logging.getLogger(__name__)
+
+
+logging_config = {
```

Comment:
You can also store the config as a file, so it does not increase the size of the build script